uint8 leaf_id
uint16 addr
uint16 value
---

uint8 exception_code
uint16 addr
uint16 value